truss/templates/server.Dockerfile.jinja

```diff
@@ -98,7 +98,7 @@ RUN uv python install {{ control_python_version }}
 RUN uv venv /control/.env --python {{ control_python_version }}
 
 COPY ./control /control
-RUN uv pip install -r /control/requirements.txt --python /control/.env/bin/python
+RUN uv pip install -r /control/requirements.txt --python /control/.env/bin/python --no-cache-dir
```

Comment:
Separate improvement I noticed, missed a `--no-cache-dir` for some installation

Reply:
lgtm